llvm/test/CodeGen/NVPTX/lower-byval-args.ll

```diff
@@ -219,6 +219,20 @@ entry:
   ret void
 }
 
+; Function Attrs: mustprogress nofree norecurse nosync nounwind willreturn memory(argmem: readwrite)
+define dso_local void @memcpy_from_param_noalign (ptr nocapture noundef writeonly %out, ptr nocapture noundef readonly byval(%struct.S) %s) local_unnamed_addr #0 {
+; COMMON-LABEL: define dso_local void @memcpy_from_param_noalign(
```

Comment:
You may also want to make this function a kernel, by adding an appropriate metadata at the end of the file.

Reply:
This is copied from line 207, simply removing `align 4` in param.

Reply:
Updated metadata to include new func.